app/assets/javascripts/discourse/app/components/bootstrap_mode_notice.js

```diff
@@ -0,0 +1,39 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import Component from "@ember/component";
```

Comment:
Oh, I'm very sorry, I missed this on my first take, but can we build this as a Glimmer component?

Reply:
No, this was a mistake on my side. My last push reverted the Glimmer component to the old one. I've now pushed the new Glimmer component. Thanks for catching this and apologies.